src/ape_ethereum/provider.py

```diff
@@ -446,7 +448,17 @@ def send_call(
 
         return HexBytes(trace.return_value)
 
-    def _eth_call(self, arguments: List) -> HexBytes:
+    def _eth_call(self, arguments: list) -> HexBytes:
+        # Force the usage of hex-type to support a wider-range of nodes.
```

Comment:
this is the only change i dont remember making as part of this PR. it seems fine though but just wondering where it came from suddenly.

Reply:
but can merge! just thought id point it out

Reply:
> Looks like it's part of the `main` branch

ah ok, we should rebase feat/08 with main first (ideally)